.github/workflows/dapr.yml

```diff
@@ -73,10 +73,8 @@ jobs:
           image: tonistiigi/binfmt:latest
           platforms: arm64
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
         if: matrix.target_os == 'linux' && github.event_name != 'pull_request'
-        with:
-          version: v0.9.1 # Don't use latest since it broke our workflow once
```

Comment:
Pin this too, even if it means a new version. We don't want to debug this again in the future.

Reply:
Done